src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class HostMetrics
+    {
+        /// <summary>
+        /// Gets or sets the instance Id.
+        /// </summary>
+        public string InstanceId { get; set; }
```

Comment:
Will the platforms metric pipeline set this already?

Reply:
I checked with Anjuna and apparently not, we will need to provide this value